Lib/tracemalloc.py

```diff
@@ -247,7 +257,8 @@ def format(self, limit=None, most_recent_first=False):
 def get_object_traceback(obj):
     """
     Get the traceback where the Python object *obj* was allocated.
-    Return a Traceback instance.
+    Return a Traceback instance
+    .
```

Comment:
Is it a deliberate chose to put the dot on a newline? If not, please revert this change.

Reply:
No sorry, PEBKAC.